core/src/storage/device.py

```diff
@@ -164,6 +164,8 @@ def load_settings(
         common.set(_NAMESPACE, _LABEL, label.encode(), True)  # public
     if use_passphrase is not None:
         common.set_bool(_NAMESPACE, _USE_PASSPHRASE, use_passphrase)
+        if use_passphrase is False:
```

Comment:
let's move this to the top level, above the "if field is set" block -- so that we don't rely on `passphrase_always_on_device` check being after `use_passphrase` check

Reply:
Fixed